### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by BhavyaSheth22**
1. Fixed logout/login buttons on navbar
2. Removed dummy data from landing page
3. Added coming soon section in prod environment

### CODE REVIEW

File: client/src/components/LoginRegister/inputOTP.jsx [hunk 1]
```diff
@@ -31,8 +31,12 @@ const OtpPage = (props) => {
           email: props.data.email,
           otp: otp
         }).then((res) => {
-          console.log(res);
-          history.push("/");
+          if (res.error) {
+            setError(res.error);
+          } else {
+            console.log(res);
```
Comment (pal-16) [inline client/src/components/LoginRegister/inputOTP.jsx:37]: We can remove if not required.

File: client/src/components/LoginRegister/inputOTP.jsx [hunk 2]
```diff
@@ -43,7 +47,7 @@ const OtpPage = (props) => {
           }
         }).then((res) => {
           if (res.error) {
-            setError("There was an error. Please try again");
+            setError(res.error);
           } else {
             console.log(res);
```
Comment (pal-16) [inline client/src/components/LoginRegister/inputOTP.jsx:52]: We can remove log if not required.

File: client/src/components/Navbar.js [hunk 3]
```diff
@@ -54,6 +55,8 @@ const Navbar = () => {
   const dispatch = useAuthDispatch();
   const history = useHistory();
 
+  console.log("prod: ", isProd);
```
Comment (pal-16) [inline client/src/components/Navbar.js:58]: We can remove log if not required.

**Review by pal-16 — APPROVED** [review]

File: client/src/components/Navbar.js (hunk 3 above)
Comment (pal-16) [reply client/src/components/Navbar.js:58]: Approving the changes as only logs issue. Rest connectivity looks good. Thank you!

File: client/src/components/Navbar.js [hunk 4]
```diff
@@ -46,6 +46,7 @@ const NavbarStyles = makeStyles({
 });
 
 const Navbar = () => {
+  const isProd = process.env.REACT_APP_ENV === "production";
```
Comment (ShubhankarKG) [inline client/src/components/Navbar.js:49]: This adds a new environment variable, we can maybe use NODE_ENV itself that React seems to use.

Source: https://medium.com/@tacomanator/environments-with-create-react-app-7b645312c09d

**Review by ShubhankarKG — CHANGES_REQUESTED** [review]

**Review by ShubhankarKG — APPROVED** [review]

**Pull request merged**